aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1209 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
```

Comment:
Pretty nit and just a stylistic choice, but I feel like the tables on this dashboard should have their columns in order of descending order/cardinality ... so like the columns should go "Job", "Aerospike cluster", "Instance", (then "Namespace" for the 1 panel with namespaces), but again a stylistic choice so your call on this

Reply:
But I did notice while making the above comment that the column names aren't sentence case and afaik I think they need/should be sentence case. So the Job, Instance, Namespace, and Usage columns may need to changed... pretty sure that can be done with an override or transformation.

Reply:
I've already showed this to Gabriel and he has yet to comment on any of these things. I'll resolve this convo for now and wait for external review here.